README.md

```
@@ -0,0 +1,14 @@
+# The ALFS Whistleblower Ticket System
+
+### A secure case management system built with Spring Boot for handling whistleblower reports.
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)